quinn-proto/src/endpoint.rs

```diff
@@ -585,6 +588,11 @@ impl Endpoint {
     pub fn accept(&mut self) {
         self.incoming_handshakes -= 1;
     }
+
+    /// Whether to unconditionally reject future incoming connections
+    pub fn busy(&mut self, state: bool) {
```

Comment:
Since we don't need to reenable here by definition, no need to take an argument? Also let's prefix the name with `set_` unless it can mirror the name of the field and that name starts with a verb?

Reply:
The high-level API will never reenable, but is that sufficient reason not to expose the capability when it's so trivial to support?

Reply:
Yes! YAGNI.

Reply:
I mean, it's public API, but sure.